credits.txt

```diff
@@ -340,7 +346,6 @@ contributed to Endless Sky:
   Ornok
   ovari
   OverYrPaygrade
-  pakyinw
```

Comment:
Question: Why was this name removed?

Reply:
That account changes username.
They authored #6174.
You can see in the [raw commit](https://github.com/endless-sky/endless-sky/commit/b8a8961f89d5430d166f2686981ba8ec322a2c41.patch) that the account previously used this now removed username, but the account that created the PR now clearly has a new name.

Reply:
Thanks for the explanation. I expected that something like that would be the case, I appreciate the confirmation.